framework/src/org/apache/cordova/CordovaWebViewImpl.java

```diff
@@ -238,7 +240,9 @@ public void showWebPage(String url, boolean openExternal, boolean clearHistory,
             } else {
                 intent.setData(uri);
             }
-            cordova.getActivity().startActivity(intent);
+            if (cordova.getActivity() != null) {
```

Comment:
If we do decide if logging debug statements is a good idea, then one should be also printed if this condition fails.